src/main/java/com/gluonhq/substrate/model/Triplet.java

```diff
@@ -58,6 +58,27 @@ public static Triplet fromCurrentOS() throws IllegalArgumentException {
         }
     }
 
+    /**
+     * @return true if host is Windows
+     */
+    public static boolean isWindowsHost() {
+        return new Triplet(Constants.Profile.WINDOWS).equals(fromCurrentOS());
```

Comment:
does this really requires the creation of a new Object?

Reply:
I could simply do:
```
String osName  = System.getProperty("os.name").toLowerCase(Locale.ROOT);
return osName.contains("windows");
```
instead?
Then we could also revert `fromCurrentOS` to use these `isXHost` methods?

Reply:
That sounds better indeed.

Reply:
ok, pushed